pkgs/applications/graphics/eddy/default.nix

```diff
@@ -0,0 +1,42 @@
+{ python37Packages
```

Comment:
Can we use a newer python?

Reply:
Resolved, used instead `python3Packages`.